pkg/kubelet/kubelet_pods.go

```diff
@@ -88,7 +88,19 @@ func (kl *Kubelet) listPodsFromDisk() ([]types.UID, error) {
 
 // GetActivePods returns non-terminal pods
 func (kl *Kubelet) GetActivePods() []*v1.Pod {
-	allPods := kl.podManager.GetPods()
+	allPods, mirrorPods := kl.podManager.GetPodsAndMirrorPods()
```

Comment:
Looking at the other usage of mirrorPods in removeOrphanedPodStatuses, the mirrorPods can be in the Map structure shown below (and not affecting functionality).
I wonder if GetPodsAndMirrorPods should return mirror pods in Map.

Reply:
@tedyu this PR is incomplete. Please check the other PR #75144.